pkg/util/parquet/writer.go

```diff
@@ -127,10 +191,15 @@ type Writer struct {
 // TODO(#99028): maxRowGroupSize should be a configuration Option, along with
 // compression schemes, allocator, batch size, page size etc
 func NewWriter(sch *SchemaDefinition, sink io.Writer, opts ...Option) (*Writer, error) {
+	schMeta, err := MakeSchemaMetadata(sch)
```

Comment:
should we always add this metdata? Or should it be only added in tests?

Reply:
Good point. I'll make it a test only thing (probably by adding a `WithCRDBReaderMeta` option). It can take an enum for any additional meta (ex. primary key cols which are needed in changefeed tests).